_clubs/naak/naak-history-and-accomplishments.md

```diff
@@ -0,0 +1,52 @@
+
+NAAK volunteers provide careful and responsible stewardship of Klahanie’s valued and diverse natural areas and the wildlife they support.
+
+### Accomplishments
+
+* Worked in partnership with King County and Puget Sound Energy to secure funding and resources to remove noxious weeds and restore critical habitats in Klahanie's major Native Growth Protection Areas (NGPAs). To date, this has resulted in over $100,000 in environmental grants and free restoration services.
+* Partnered with Puget Sound Energy (PSE) to restore and enhance six NGPA sites. These mitigations included removal of noxious weeds, the addition of hundreds of native plants and free maintenance and watering for three years until the plants become self-sustaining.  A portion of the Mountain View natural area was completed in January, 2018.  Scotch broom and Himalayan blackberries were removed and 127 native plants were added, including Pacific crabapples, Red-flowering currants, Snowberries and Nootka roses.  In 2019, we look forward to partnering with Bonneville Power Administration (who has assumed responsibility for the maintenance of the power line corridor from PSE) in pursuing additional environmental mitigation opportunities.
+* Coordinated efforts between King County and Klahanie to ensure mandated noxious weeds were removed, according to strict County requirements. This includes the ongoing removal of Garlic mustard and Purple loosestrife.  In addition, volunteer work parties have removed several thousand Tansy ragwort plants, which is highly invasive and can be poisonous to animals.
+* Obtained and coordinated the services of, and worked along with, EarthCorps to remove a vast number of noxious weeds, including the removal of destructive English ivy from hundreds of trees.
+* A variety of native plants and trees compatible with each unique natural habitat have been planted.  One example: The Yellow Lake area restoration included adding trees to "shade out" noxious weeds and replace older trees as they die.
+* Our ongoing collaboration has reduced noxious weeds to a small fraction of the overall vegetative coverage in Klahanie’s native growth areas.  And, through ongoing stewardship, re-growth is being controlled.
+* Sponsored dozens of work parties throughout the years to remove noxious weeds, such as English ivy, Scotch broom, and Tansy ragwort from natural areas throughout Klahanie. The progress in removing and controlling these weeds continues.  Today, Klahanie’s natural areas are in a state of stewardship where major removal efforts have been drastically reduced and the focus is now on control measures.  In addition, several work parties included the participation of the Eagle Scouts.  These work parties focused on restoring a sensitive area in the Yellow Lake NGPA that had been destroyed by off trail biking.

```

Comment:
I'm not taking the time to edit this long list right now, but these two bullet points do not use the same format (verb first) as all the others. For consistency, I'd rewrite these two. Ex: "Planted a variety of native plants and trees compatible with each unique natural habitat." "Collaborated to reduce noxious weeds to a small fraction..."